docs/modules/user-guide/partials/proc_adding-commands-to-a-devfile.adoc

```diff
@@ -17,7 +17,7 @@ A devfile allows to specify commands to be available for execution in a workspac
 
 . For each command, define an unique value for the mandatory `id` attribute.
 
-. For each command, define a mandatory type of one of the following types: `exec` or `vscode-tasks`.
+. For each command, define a mandatory type of one of the following types: `exec`, `apply`, `composite`, or `vscode-tasks`.
```

Comment:
Is vscode-tasks still a valid type? I ask, without researching the issue, because you removed significant content related to vscode-tasks further down.

Reply:
yes its still a valid type as of now but to be removed in the next release.